style.css

```diff
@@ -364,3 +364,77 @@ li {
   font-weight: 500;
   font-size: 20px;
 }
+
+.contact {
+  background-color: #6070ff;
+  background-image: url(./img/contact\ form\ shapes-mobile.png);
+  background-position: right;
+  background-repeat: no-repeat;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  border-top-left-radius: 100px;
+}
+
+.contact h2 {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  text-align: center;
+  color: #fff;
+  padding-top: 103px;
+  padding-bottom: 12px;
+}
+
+.contact p {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  color: #ebebff;
+  text-align: center;
+  padding: 0 24px 30px 24px;
+}
+
+#username,
+#user-email,
+#text-area {
+  width: 90%;
+  margin: 20px 24px;
+  height: 2rem;
+  border-radius: 8px;
+  border: 1px solid #cfd8dc;
+}
+
+#text-area {
+  resize: none;
+  height: auto;
+}
+
+.form-btn {
+  width: 140px;
+  height: 48px;
+  margin-left: 24px;
+  margin-bottom: 45px;
+  border-radius: 8px;
+  background: #fff;
+  align-items: center;
+  padding: 12px 16px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+  border: 1px solid #cfd8dc;
+}
+
+.form-btn:hover {
+  color: #fff;
+  background-color: #6070ff;
+  cursor: pointer;
```

Comment:
- [ ] Kindly add the active and pressed effect to the button to improve the user experience :factory_worker: